### ISSUE TITLE
Linux start script fails when symbolic link

### ISSUE THREAD

**Issue by miurahr** [amake/SuperTMXMerge#3]
When installing SuperTMXMerge on `/opt/SuperTMXMerge/` and making symbolic link
`ln -s /opt/SuperTMXMerge/SuperTMXMerge.sh /usr/local/bin/supertmxmerge`
It fails.

It should be changed like followings:
```
SCRIPT_DIR=$(dirname "$(readlink -f $0)")
java -jar "$SCRIPT_DIR/SuperTMXMerge.jar" "$@"
```

With this, script can follow symbolic link and start properly.

**Comment by amake** [comment]
Thanks for the report. `readlink` is not available on OS X, but I suppose it's not unreasonable to use it since OS X has a separate .app bundle.

**Comment by amake** [comment]
Fixed in v1.2.
